src/main/java/org/victorrobotics/frc/dtlib/logging/DTLog.java

```diff
@@ -0,0 +1,61 @@
+package org.victorrobotics.frc.dtlib.logging;
+
+import org.victorrobotics.frc.dtlib.DTSubsystem;
+
+import java.io.BufferedOutputStream;
+import java.io.DataInputStream;
+import java.io.DataOutputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DTLog {
```

Comment:
Let's keep this as-is for now. As we move forward with data logging and dashboarding, we will revisit.

Reply:
The logging is just a prototype that I generated at 1 am last night, it is by no means permanent.